fontend/app/products/page.tsx

```
@@ -391,10 +1153,14 @@ const ProductList = () => {
                                         )}
                                         <p className="current-price">{formatPrice(product.minPrice)}</p>
                                     </div>
+                                    <button className="add-to-cart-button">THÊM VÀO GIỎ</button>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)